contracts/PetPark.sol

```diff
@@ -3,5 +3,58 @@ pragma solidity ^0.8.0;
 
 
 contract PetPark {
+    enum AnimalType{None,Fish,Cat,Dog,Rabbit,Parrot}
+    struct PetOwnerDetails{
+        AnimalType pet;
+        uint age;
+        Gender sex;
+    }
+    mapping(address => PetOwnerDetails)petOwner;
+    mapping(AnimalType => uint)public animalCounts;
+
+    enum Gender{Male,Female}
+    address owner;
+    event Added(AnimalType addedAnimal,uint count);
+    event Borrowed(AnimalType borrowedAnimal);
+    constructor(){
+        owner = msg.sender;
+    }
+    function add(AnimalType _animal, uint _count)public {
+        require(owner == msg.sender,"Not an owner");
```

Comment:
Better to use a modifier here to check for owner for better code readability

Reply:
I had a doubt when to use simple require and when to use modifier? Like if some condition asked more than once then I should automatically go for modifier?

Reply:
Yes, for anything you feel you may use more than once or when it provides better code readability